shelley/design-spec/delegation_design_spec.tex

```diff
@@ -461,31 +460,30 @@ \subsection{Overview of Delegation}
     %% Nodes
     %%
     \node (owner)   [concept]                    {Owner};
-    \node (addr)    [entity, left = of owner]    {Pubkey address};
-    \node (skey)    [entity, below = of owner]   {Staking key};
-    \node (pkey)    [entity, left = of skey]     {Payment key};
-    \node (racc)    [entity, right = of skey]    {Reward account /
-      Reward address};
+    \node (valaddr)    [entity, left = of owner]    {Value address};
+    \node (registeredStakeAddr)    [entity, below = of owner]   {Registered stake address};
```

Comment:
the staking object can also be of key or script type, do you want to reflect this analogously in the diagram?

Reply:
So should this be "registered stake object"? (instead of "registered stake address"?)

Reply:
maybe use registered stake address which then `contains` a key or script as above